solutions/src/main/java/ru/sberbank/school/task02/ServiceFactoryImpl.java

```diff
@@ -0,0 +1,13 @@
+package ru.sberbank.school.task02;
+
+public class ServiceFactoryImpl implements ServiceFactory{
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1208686759>*